ocs_ci/helpers/helpers.py

```diff
@@ -3480,3 +3480,17 @@ def wait_for_rook_ceph_pod_status(pod_obj, desired_status, timeout=420):
             return False
 
     return True
+
+
+def check_mon_pods_eq_3():
+    """
+    Function to check the number of monitoring pods
+
+    Returns:
+        bool: True if number of mon pods is 3, False otherwise
+
+    """
+    mon_pod_list = pod.get_mon_pods()
+    if len(mon_pod_list) == 3:
+        return True
+    return False
```

Comment:
Before return false, also log the len of mon pods and the mon pod names. this will help during test failures, without which it can be difficult to debug.

Reply:
You have changed the return bool values which might break tests/manage/z_cluster/test_rook_operator_restart_during_mon_failover.py Please check and make necessary changes to this test if required

Reply:
Added, Also modified the test_rook_operator_restart_during_mon_failover.py file accordingly